megamek/src/megamek/common/Entity.java

```diff
@@ -2574,7 +2574,7 @@ public boolean isOnAtmosphericGroundMap() {
      * @return True if this is an airborne aircraft on a ground map.
      */
     public boolean isAirborneAeroOnGroundMap() {
-        return isAero() && isAirborne() && getGame().getBoard().onGround();
+        return isAero() && isAirborne() && getGame() != null && getGame().getBoard().onGround();
```

Comment:
Rather than adding a bunch of null checks, do we want to think about creating a new lightweight Game class just for AI training?

Reply:
Not rather, but would be nice to have a lightweight game.
Entity today expects that `Game` can be null in some situations, but does not handle `null` in all the rest, we need to make sure that either it is impossible to have Entity without a reference to Game, or make Entity be able to exists witohut `Game`.